app/jobs/notify_subscriber.rb

```diff
@@ -13,17 +13,24 @@ def self.perform_later(subscriber_id, submission)
   def perform(subscriber_id, submission)
     raise_error = false
     subscriber = Subscriber.find(subscriber_id)
-    raise_error = handle_failure(subscriber) unless send_message_to_webhook(subscriber.webhook_url, submission)
+    unless send_message_to_webhook(subscriber.webhook_url, submission)
+      raise_error = handle_failure_and_decide_whether_to_raise_error(subscriber)
```

Comment:
`handle_failure_maybe_raise`?

Reply:
Ooh that's nicer.

Reply:
Naming things is hard, I do kinda wish Ruby had some kind of standard documentation thing like JSDoc